enterprise/server/ociregistry/ociregistry.go

```diff
@@ -245,22 +256,49 @@ func (r *registry) handleBlobRequest(ctx context.Context, w http.ResponseWriter,
 		}()
 	}
 
+	casDigest := &repb.Digest{
+		Hash:      blobSHA256Digest,
+		SizeBytes: blobSize,
+	}
+	bsClient := r.env.GetByteStreamClient()
+	resourceName := digest.NewResourceName(
```

Comment:
github lost my comment somehow... you should enable compression for both the upload and download paths by calling `resource.SetCompressor(repb.Compressor_ZSTD)`.